e2e-patient-test/src/test/java/gov/cms/ab2d/SpringBootApp.java

```diff
@@ -0,0 +1,12 @@
+package gov.cms.ab2d;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SpringBootApp {
```

Comment:
Do we really need this to run the e2 tests?

Reply:
So, it's not technically e2e as much as it requires outside resources (BFD sandbox) to run. It auto-wires the BFD client. I wanted it separate from the bfd module.